Pipfile

```diff
@@ -0,0 +1,41 @@
+[[source]]
+url = "https://pypi.python.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+Django = "*"
+django-ldapdb = "*"
+bleach = "*"
+python-creole = "*"
+requests = "*"
+oauthlib = "*"
+urllib3 = "*"
+certifi = "*"
+idna = "*"
+chardet = "*"
+django-auth-ldap = "*"
+django-oauth-toolkit = "*"
+django-cors-headers = "*"
+django-widget-tweaks = "*"
+etherpad_lite = "*"
+fuzzywuzzy = "*"
+nameparser = "*"
+networkx = "*"
+psycopg2-binary = "*"
+pygpgme = "*"
+pygraphviz = "*"
+pytz = "*"
+PyPDF2 = "*"
+raven = "*"
+pyldap = "*"
+python-magic = "*"
+pydiscourse = "*"
+django-email-log = "*"
+Pillow = "*"
+python-Levenshtein = "*"
+
+[dev-packages]
+
+[requires]
+python_version = "3.7"
```

Comment:
For consistency across our projects (cf. Bitpoll) I would have preferred to use `piptools sync` (which additionally has a functionality to lock hashes of packages just like the `Pipfile.lock`) rather than introducing another alternative pip wrapper.

However, as the work has been done already, I am fine with keeping this additional tool anyway (in the hope that the next merge request on another project does not add yet another pip wrapper …)

Reply:
Huh, I didn't even know that existed. Pipenv is just the tool which Timon and I have started to use :)